action-server/tests/lib/test_phone_number_validation.py

```diff
@@ -0,0 +1,8 @@
+import unittest
+
+from actions.lib.phone_number_validation import generate_validation_code
+
+
+class PhoneNumberValidationTest(unittest.TestCase):
+    def test_generate_validation_code(self):
+        assert len(generate_validation_code()) == 4
```

Comment:
And validate that it's only numbers? The dialogue will expect it... If it is not I'll have to make some unpleasant changes...

Reply:
Sure 😄